.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# https://help.github.com/en/articles/about-code-owners
+# Default reviewers for everything
+* @i386x @pcahyna @richm
```

Comment:
@seocam you should add yourself here and remove i386x

Reply:
ok!

Reply:
https://github.com/linux-system-roles/certificate/pull/1/commits/7369b739334fbdc60be814795b2dbbffae51533b